docs/network-flow-visibility.md

```diff
@@ -217,6 +248,10 @@ kubectl delete -f https://raw.githubusercontent.com/antrea-io/theia/main/build/c
 
 ### Configuration
 
+So far, we went through Grafana Flow Collector deployment with the default
+configuration. The Grafana Flow Collector also exposes a few configuration
```

Comment:
few configuration -> few configurations

Reply:
We have "parameters" after "configuration": "a few configuration parameters".